src/types/Ctype.ts

```diff
@@ -25,20 +33,19 @@ export interface IClaimInput {
   title: string
   description?: string
   type: string
+  owner?: string
 }
 
 export class CType implements ICType {
-  public static fromObject(obj: ICType): CType {
+  public static fromObject(obj: CTypeMetadata): CType {
```

Comment:
Can we get rid of this whole class, or would it be too much?

Reply:
Yes this is possible, it's functionality is only used in a few places, I removed it for now and used our old method of acquiring the property title and extended it with an additional check.

```
cType && cType.metadata.properties[propertyName].title.default ?
      cType.metadata.properties[propertyName].title.default : propertyName
```